integration_tests/gatway_fqdn_test.go

```diff
@@ -117,7 +110,6 @@ func TestGatewayFQDNDeployment(t *testing.T) {
 	err = tfPluginClient.NetworkDeployer.Cancel(ctx, &network)
 	assert.NoError(t, err)
 
-	_, err = tfPluginClient.State.LoadGatewayFqdnFromGrid(nodeID, gw.Name)
```

Comment:
Let's make the function name `LoadGatewayFQDNFromGrid` instead of `LoadGatewayFqdnFromGrid`